Firestore/Source/Public/FIRGetOptions.h

```diff
@@ -62,7 +65,7 @@ typedef NS_ENUM(NSUInteger, FIRGetSource) {
 /**
  * Initializes the get options with the specified source.
  */
-- (instancetype)initWithSource:(FIRGetSource)source NS_SWIFT_NAME(init(source:));
++ (instancetype)source:(FIRGetSource)source;
```

Comment:
Methods should lead with their return type in Objective-C

This should be `optionsWithSource:`